src/Modal/Modal.jsx

```diff
@@ -79,5 +79,5 @@ Modal.propTypes = {
 
 Modal.defaultProps = {
   width: DEFAULT_WIDTH,
-  focusLocked: false,
+  focusLocked: true,
```

Comment:
note: if we do default to true, this should be a major bump to `0.26.0` so that clients don't pull it in automatically

Reply:
Thanks, I'll make the change